test/TestUtilities/Test.Utility/TestPackages.cs

```diff
@@ -282,7 +283,7 @@ public static FileInfo GetEmptyPackageWithDependencies(string path, string packa
             StringBuilder sb = new();
             for (int i = 0; i < 10000; i++)
             {
-                sb.AppendLine(i.ToString());
+                sb.AppendLine(i.ToString(CultureInfo.CurrentCulture));
```

Comment:
You can append the integer directly

```suggestion
                sb.AppendLine(i);
```

Reply:
Unfortunately, AppendLine only accepts a string